tests/test_scala.py

```diff
@@ -0,0 +1,29 @@
+def test_java_exists(Command):
```

Comment:
Addressed in [99ecbdc](https://github.com/azavea/ansible-scala/pull/3/commits/99ecbdc317b7d946df54419288b9902daec13154)